blog/Getting Started with ANP.md

```diff
@@ -0,0 +1,274 @@
+the AdminNetworkPolicy API provides two cluster-level resources that can not only enforce
+inviolable policy rules (for traffic that should always be allowed or denied), but also
+provide cluster default guardrails that can be overridden by namespace users if needed.
+
+Follow along with the example below, as we showcase how the AdminNetworkPolicy API achieves
+this, and how it interoperates with NetworkPolicy objects to provide a complete,
+priority-based policy enforcing solution in a K8s cluster.
+
+![Diagram1](Diagram1.png)
+<p style="text-align: center"><b>Tenants and System Namespaces</b></p>
+
+Imagine we have a bare-bones cluster set up by a company's IT team, which only has a
+handful of namespaces. The sales team owns two namespaces `s1` and `s2` (both labeled
+tenant=sales), which runs web and db applications respectively. On the other hand,
+the accounting team owns a single namespace `a1`, which, for demonstration purposes,
+only has a single pod deployed. In addition, there is a `monitoring-ns` running an
+application that needs to probe into namespaces required by compliance team. Of course
+there is also `kube-system` who provides DNS services for the cluster via its `kube-dns`
+pods.
+
+With the help of the new API, admins can easily implement the networking requirements
+mentioned above, namely isolating pods in different tenants by default and ensuring all
+pods are able to make DNS requests and accepting monitoring probes.
+
+The syntax for the AdminNetworkPolicy API is very similar to that of the current network
+policy API. However, there are some traits of the AdminNetworkPolicy API that are very
+different and worth highlighting:
+
+- `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` are cluster-scoped resources.
+  The subject of the policy can be the entire cluster namespaces, a select set of namespaces,
+  or even select set of pods in a select set of namespaces, depending on the actual use of
+  the policy.
+
+- AdminNetworkPolicy rules are enforced in a hierarchical order based on priority.
+  `AdminNetworkPolicy` objects with a smaller numerical `priority` in its spec have higher
+  precedence over those with a larger numerical `priority`. Furthermore, rules inside a
+  policy are enforced in the order they are written (ingress and egress rules does not have
+  relevant order among each other though). To avoid potential conflict in rules, we generally
+  advise against creating multiple `AdminNetworkPolicy` objects at the same `priority`.
+
+- AdminNetworkPolicy rules should be interpreted as-is: pod-to-pod traffic only matches the
+  rule if all criteria listed in the rule are matched. That means src and dst of the packet
+  matches subject and ingress/egress peer selection, and in addition port/protocol also
+  matches, if those are specified in the rule. If so, `action` of the rule makes final verdict
+  on whether the packet is allowed, dropped, or selected to skip any further policy rule
```

Comment:
perhaps the ability to allow/drop/skip should be the first phrase of its own bullet point? To me, that seems a critical difference since NetPolV1 rules only allow (and implicitly deny)

Reply:
Done